src/main/java/core/basesyntax/strategy/impl/TransactionStrategyImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.strategy.AmountHandler;
+import core.basesyntax.strategy.TransactionStrategy;
+import java.util.Map;
+
+public class TransactionStrategyImpl implements TransactionStrategy {
```

Comment:
great job! let's only rename it, so it is more obvious what is the purpose of this method
```suggestion
public class AmountStrategyImpl implements AmountStrategy {
```